### DISCUSSION TITLE
Integration with Atlassian Statuspage

### DISCUSSION THREAD

**Discussion by guillaumemeyer** [hyperjumptech/monika#830] [Ideas]
Would be really great to be able to send alerts to [Atlassian Statuspage](https://www.atlassian.com/software/statuspage).

Following the [documentation](https://support.atlassian.com/statuspage/docs/know-when-to-automate-your-status-page/), it could be done through API or through a simple email.
Email integration would be easier, but it requires specific keywords `UP` and `Down` in the subject line, and as far as I can understand, we can't customize Monika's alert emails...

**Comment by guillaumemeyer** [comment]
To make it clear, I know we can define some parts of the alert message using a template. The problem is that you can't define a distinct and specific subject line for incident and recovery.

**Reply by nicnocquee** [reply]
Hello @guillaumemeyer , good idea, let us explore the Atlassian documentation first.

**Comment by guillaumemeyer** [comment]
Great to have this integration @haricnugraha .
@nicnocquee Is there a way to associate a specific probe in Monika to a component id in Statuspage?
Cause right now whatever the failing probe, Monika just posts a global "Service is down" message...
